From 29c65ded8fc6f1ab8329a7da49ba1ff39b678a43 Mon Sep 17 00:00:00 2001
From: Tim Gardner <tim.gardner@canonical.com>
Date: Thu, 18 Oct 2012 12:30:20 -0600
Subject: [PATCH 0037/1767] UBUNTU: SAUCE: (no-up) fix highbank include path

Continue to carry as its not hurting anything. Of course I could just
build test without this patch, but thats a giant pain.

Signed-off-by: Tim Gardner <tim.gardner@canonical.com>
---
 arch/arm/mach-highbank/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/mach-highbank/Makefile b/arch/arm/mach-highbank/Makefile
index 55840f4..21e008d 100644
--- a/arch/arm/mach-highbank/Makefile
+++ b/arch/arm/mach-highbank/Makefile
@@ -1,3 +1,5 @@
+KBUILD_CFLAGS += -I$(srctree)/arch/arm/mach-highbank/include
+
 obj-y					:= highbank.o system.o smc.o
 
 plus_sec := $(call as-instr,.arch_extension sec,+sec)
-- 
1.9.3

